L1Trigger/DTTriggerPhase2/src/MuonPathAssociator.cc

```diff
@@ -602,6 +625,8 @@ void MuonPathAssociator::correlateMPaths(edm::Handle<DTDigiCollection> dtdigis,
               double x_inSL1 = SL3metaPrimitive->x + SL3metaPrimitive->tanPhi * (VERT_PHI1_PHI3 - l_shift);
               for (auto digiIt = (dtLayerId_It.second).first; digiIt != (dtLayerId_It.second).second; ++digiIt) {
                 DTWireId wireId(dtLId, (*digiIt).wire());
```

Comment:
Can this be moved after the `if` here below?

Reply:
But then `x_inSL1` will have to be computed in (possibly) every iteration of the loop, is that desired?